java/java.lsp.server/vscode/src/jdk/validation/extensionUtils.ts

```diff
@@ -0,0 +1,18 @@
+import * as vscode from 'vscode';
+import { client as nblsClient } from '../../extension';
+
+const RH_EXTENSION_ID = 'redhat.java';
+
+export function isRHExtensionActive(): boolean {
+    const rh = vscode.extensions.getExtension(RH_EXTENSION_ID);
+    return rh ? true : false;
+}
+
+export async function waitForNblsCommandToBeAvailable(command: string) {
+    await nblsClient;
+    const commands = await vscode.commands.getCommands();
+
+    if (!commands.includes(command)) {
```

Comment:
We no longer need the check for a particular command .... the client will either start and the promise fullfills, or the client fails, the promise fais - which causes `await nblsClient` to throw an error on itself. If the client initializes, but fails to register the command for whatever reason, vscode's `commands.executeCommand()` will throw an error.  So no other explicit throw is necessary, IMHO.